crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -533,11 +532,18 @@ impl<T: R2D2Connection + 'static> PgIndexerStore<T> {
                     stored_checkpoints.chunks(PG_COMMIT_CHUNK_SIZE_INTRA_DB_TX)
                 {
                     insert_or_ignore_into!(checkpoints::table, stored_checkpoint_chunk, conn);
-                    let time_now_ms = chrono::Utc::now().timestamp_millis();
+                    let system_time_now = SystemTime::now();
                     for stored_checkpoint in stored_checkpoint_chunk {
-                        self.metrics
-                            .db_commit_lag_ms
-                            .set(time_now_ms - stored_checkpoint.timestamp_ms);
+                        system_time_now
```

Comment:
basically impl. of report_checkpoint_age_ms as CheckpointSummary does not go this far, and we only have indexed checkpoint here.